extensions/amp-redbull-player/validator-amp-redbull-player.protoascii

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-redbull-player
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-redbull-player"
+    version: "0.1"
+    version: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-redbull-player>
+  html_format: AMP
+  tag_name: "AMP-REDBULL-PLAYER"
+  requires_extension: "amp-redbull-player"
+  attrs: {
+    name: "data-videoid"
+    mandatory: true
+  }
+  attrs: {
+    name: "data-skin"
+    mandatory: true
+  }
+  attr_lists: "extended-amp-global"
+  amp_layout: {
+    supported_layouts: FIXED
+    supported_layouts: FLEX_ITEM
+    supported_layouts: RESPONSIVE
```

Comment:
The component implementation uses `isLayoutSizeDefined` for the runtime check, which allows all of the following:

```js
  return (
    layout == Layout.FIXED ||
    layout == Layout.FIXED_HEIGHT ||
    layout == Layout.RESPONSIVE ||
    layout == Layout.FILL ||
    layout == Layout.FLEX_ITEM ||
    layout == Layout.FLUID ||
    layout == Layout.INTRINSIC
  );
```

Is there any reason why you're allowing only three layouts and not this entire set? Usually if you can support `responsive` and `fixed` you can support the rest.

Reply:
updated amp_layout